docs/standalone_refactoring_analysis.md

```
@@ -0,0 +1,1954 @@
+  SELECT "erc20Wrapper", 'CrcV2_ERC20Wrapper', avatar FROM "CrcV2_ERC20WrapperDeployed"
+) WHERE tokenAddress IN (
+  SELECT DISTINCT "to" FROM "CrcV1_Transfer"
+  UNION SELECT DISTINCT "to" FROM "CrcV2_TransferSingle"
+  UNION SELECT DISTINCT "to" FROM "CrcV2_TransferBatch"
+  UNION SELECT DISTINCT "to" FROM "CrcV2_Erc20WrapperTransfer"
+)
+```
+
+**Regression Test Result**:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)